### ISSUE TITLE
Dynamic b-tab generation doesn't work with fetch call

### ISSUE THREAD

**Issue by Magenta94** [bootstrap-vue/bootstrap-vue#3395]
### Describe the bug

On the created() hook, the result of a fetch call is saved as array of integers in the model. On the view, a v-for creates a b-tab for each element in the array. This was working in 2.0.0-rc.20

### Steps to reproduce the bug

Use Bootstrap-Vue version 2.0.0-rc.21, see demo link.

### Expected behavior

A list of tab should be created on the view after the array is populated.

### Versions

**Libraries:**

- BootstrapVue: 2.0.0-rc21
- Bootstrap: 4.3.1
- Vue: 2.6.10

**Environment:**

- Device: PC
- OS: Windows 10
- Browser: Chrome
- Version: 74

### Demo link
rc.20 - **Working**
https://jsfiddle.net/jvmp90Ln/
rc.21 - **Not working**
https://jsfiddle.net/n1hbsmpw/

**Comment by tmorehouse** [comment]
We updated the method that tabs are detected in rc.21 to fix a bug when tabs are in in lazy modals.

Manually running the internal method `updateTabs()` after mount fixes your issue, but is not an ideal workaround.

We have discovered that `this.$slots.default` does not always reflect the same child Vue instances that `this.$children` does with Vue 2.6.x (they have changed how slots are handled internally, caching some values).

Unfortunately the order of the tabs in `this.$children` does not always reflect the order of the children in `this.$slots.default` or the DOM when tabs are dynamically inserted / removed.

We might be able to rely on `this.$children` before mount (i.e. the `created` hook), but definitely can't rely on it after mount.
